cypress/e2e/tests/migration/applicationinventory/assessment/copy_assessment/copy_assessment.test.ts

```diff
@@ -16,7 +16,6 @@ limitations under the License.
 /// <reference types="cypress" />
 
 import {
-    hasToBeSkipped,
     login,
     preservecookies,
```

Comment:
Remove preservecookies  and deleteApplicationTableRows

Reply:
Also move after method to the end after the tests ,remove unnecessary comments from after method and use deleteByList() to delete everything .

Reply:
Done